src/Compilers/Test/Utilities/VisualBasic/TestOptions.vb

```diff
@@ -57,4 +57,15 @@ Friend Module TestOptionExtensions
     Public Function WithIOperationFeature(options As VisualBasicParseOptions) As VisualBasicParseOptions
         Return options.WithFeatures(options.Features.Concat(New KeyValuePair(Of String, String)() {New KeyValuePair(Of String, String)("IOperation", "true")}))
     End Function
+
+    <Extension()>
+    Public Function WithImplicitDefaultOptionalParameter(options As VisualBasicParseOptions) As VisualBasicParseOptions
+        Dim flag = Feature.ImplicitDefaultValueOnOptionalParameter.GetFeatureFlag
+        Return options.WithFeatures(options.Features.Concat(New KeyValuePair(Of String, String)() {New KeyValuePair(Of String, String)(flag, "true")}))
```

Comment:
@AlekseyTs  Does the `WithStrictFeature` also need changing to use `withExperimental`?